packages/stripe/lib/src/stripe.dart

```diff
@@ -117,6 +117,21 @@ class Stripe {
     return isSupported;
   }
 
+  /// Creates a single-use token that represents an Apple Pay credit card’s details.
+  ///
+  /// Tokens are considered legacy, use [PaymentMethod] and [PaymentIntent]
+  /// instead.
+  /// Throws an [StripeError] in case createToken fails.
+  Future<TokenData> createApplePayToken(Map<String, dynamic> payment) async {
```

Comment:
but in this case token is not legacy right? maybe we should point instead to the stripe Apple Pay method